delfin/task_manager/scheduler/schedulers/telemetry/performance_collection_handler.py

```diff
@@ -108,3 +108,6 @@ def _handle_task_failure(self, start_time, end_time):
                        FailedTask.retry_count.name: 0,
                        FailedTask.executor.name: self.executor}
         db.failed_task_create(self.ctx, failed_task)
+        self.metric_task_rpcapi.assign_failed_job(self.ctx,
```

Comment:
This message will help to assign failed job in same executor.. But with removal failed_job distributor scanning there no way to assign failed_job in case of a node down/restart .

Reply:
Ok, so we should add another mechanism to recover the failed job which belongs to this executor